addons/rose/addon/styles/rose/components/_icon.scss

```diff
@@ -3,6 +3,10 @@
 .rose-icon {
   --size: #{sizing.rems(l)};
 
+  &.extra-large {
```

Comment:
can this be replaced by `sizing.rems(l)` or `sizing.rems(xl)` or `sizing.rems(xxl)` for consistency?

Reply:
If we are to follow the wireframe exactly, the `xl` sizing was too big at 36x36. The wireframe has it sized at 32x32. I am happy to change it for consistency if needed though.

Reply:
Gotcha! Let's leave it as is.